README.md

```diff
@@ -52,50 +43,38 @@ _NOTE:_
 - The measurement of TEE, Guest's boot, OS is per CC VM, but cluster/container measurement
 might be per cluster/namespace/container for cloud native architecture.
 - Please refer structure [`TDREPORT`](https://github.com/tianocore/edk2/blob/master/MdePkg/Include/IndustryStandard/Tdx.h)
+- CCNP project collects container level primitives by implementing unified APIs defined in [CC Trusted API](https://github.com/cc-api/cc-trusted-api). The project will be moved to [CC Trusted API](https://github.com/cc-api/cc-trusted-api) in the near future. 
 
 
-## 2. Design
-
-`CCNP` includes several micro-services as BaaS(Backend as a Service) to provides
-cloud native measurement, then exposes `CC trusted API` via cloud native SDK:
-
-- Services are designed to hide the complexity of different TEE platforms and provides
-common interfaces and scalability for cloud-native environment to address the fetching
-the fetching of quote, measurement and event log.
+## 2. Installation
 
-- SDK is provided to simplify the use of the service interface for development,
-it covers communication to the service and parses the results from the services.
-With such SDK, users can perform related actions with one simple API call.
+### 2.1 Configuration
 
-- A `CCNP` device plugin is provided as the dependency for services such as Quote
-Server and Measurement Server. It will help with device mount and folder injection
-within the service.
+CCNP collects primitives of confidential cloud native environments running in confidential VMs, such as Intel® TD. You can setup an Intel® TDX enlightened host and then boot TD guest on it. The feasible configurations are as below.
```

Comment:
I don't think  Intel((r) TD is a valid product name. I could not find it in the brands database.

Reply:
Also it should be   "then boot a TD guest"